prov/rxm/src/rxm_cq.c

```diff
@@ -945,6 +945,10 @@ void rxm_ep_progress_one(struct util_ep *util_ep)
 	struct rxm_ep *rxm_ep =
 		container_of(util_ep, struct rxm_ep, util_ep);
 
+	if (rxm_ep->util_ep.domain->data_progress == FI_PROGRESS_MANUAL &&
```

Comment:
Can we remove `rxm_ep->util_ep.domain->data_progress == FI_PROGRESS_MANUAL` here? I think, it would be ok to just check `!slistfd_empty(&rxm_ep->msg_eq_entry_list)` for both manual and auto (always false) progress.

The same is applicable for `rxm_ep_progress_multi`.

Reply:
Yes, I think we can drop the redundant check.